client/src/help/HelpRelease.tsx

```diff
@@ -0,0 +1,224 @@
+ * limitations under the License.
+ */
+
+/**
+ *  renku-ui
+ *
+ *  Help.present.js
+ *  Presentational components for help.
+ */
+
+import React, { Fragment } from "react";
+import { Row, Col } from "reactstrap";
+
+import { Loader } from "../components/Loader";
+import { ExternalLink } from "../components/ExternalLinks";
+import {
+  useGetCoreVersionsQuery,
+  useGetNotebooksVersionsQuery,
+} from "../features/versions/versionsApi";
+
+type Params = {
+  RENKU_CHART_VERSION: string;
+  UI_VERSION: string;
+};
+
+function componentDocsUrl(
+  componentUrl: string,
+  taggedVersion: string | undefined,
+  devHash: string | undefined | null
+) {
+  const releasesUrl = `${componentUrl}/releases/`;
+  return taggedVersion == null
+    ? releasesUrl
+    : devHash == null
+    ? `${componentUrl}/releases/tag/${taggedVersion}`
+    : releasesUrl;
+}
+
+/**
+ * Parse the chart version into components according to https://github.com/jupyterhub/chartpress/tree/1.3.0
+ * NB This may need to change if chartpress is updated.
+ * @param version the chart version
+ * @returns the version components
+ */
+function parseChartVersion(version: string | undefined) {
+  if (version == null) {
+    return {
+      taggedVersion: "unknown",
+      devHash: "unknown",
+    };
+  }
+  const versionComponents = version.split("-");
+  if (versionComponents.length === 1) {
+    return {
+      taggedVersion: version,
+      devHash: null,
+    };
+  }
+  const taggedVersion = versionComponents[0];
+  const versionMetadata = versionComponents[1];
+  const metadataComponents = versionMetadata.split(".");
+  const lastComponent = metadataComponents[metadataComponents.length - 1];
+  if (lastComponent.match(/^h.+$/)) {
+    // This is non-tagged version, e.g. [tag].n[number of commits].h[commit hash]
+    const devHash = lastComponent.slice(1);
+    return {
+      taggedVersion,
+      devHash,
+    };
+  }
+  // This is a tagged dev version, e.g. [tag]-[build info]
+  const devHash = lastComponent;
+  return {
+    taggedVersion,
+    devHash,
+  };
+}
+
+type ComponentVersionsProps = {
+  componentUrl: string /* URL of the component's repository, without a trailing slash */;
+  isFetching: boolean;
+  taggedVersion: string | undefined;
+};
+function ComponentVersion({
+  componentUrl,
+  isFetching,
+  taggedVersion,
+}: ComponentVersionsProps) {
+  if (isFetching) {
+    return <Loader inline size={16} />;
+  }
+  const releaseUrl =
+    taggedVersion != null
+      ? `${componentUrl}/releases/tag/${taggedVersion}`
+      : `${componentUrl}/releases/`;
+  return (
+    <>
+      <ExternalLink
+        role="text"
+        title={taggedVersion ?? "unknown"}
+        url={releaseUrl}
+      />
+    </>
+  );
+}
+
+function CoreRelease() {
+  const { data, isFetching } = useGetCoreVersionsQuery();
+  return (
+    <ComponentVersion
+      componentUrl="https://github.com/SwissDataScienceCenter/renku-python"
```

Comment:
Let's keep the renku repositories URL in a single place. Check out /utils/constants/Repositories.ts

```suggestion
      componentUrl={RenkuRepositories.Python}
```